crmsh/constants.py

```diff
@@ -490,4 +490,16 @@
 
 
 SSH_OPTION = "-o StrictHostKeyChecking=no"
+
+
+PROFILES_DICT = {
+        "Azure": {
```

Comment:
It would be good to have this `Azure` as a constant, or even some kind of enum, as it is used as string in other places, and there are chances of doing some error if the string is changed